### PR TITLE
Teacher Assignment 

### PR DESCRIPTION
**Pull request by fuelupeducation**
Need to check

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -5,5 +5,10 @@
   </head>
   <body>
     <div id="app"></div>
+    <script
```
Comment (ArchitW) [inline src/index.html:8]: why are we adding script via src this is node project, can we install it?

File: src/pages/Landing/components/Body/Body.module.scss [hunk 2]
```diff
@@ -0,0 +1,4 @@
+.Body {
+  background-color: yellowgreen;
```
Comment (ArchitW) [inline src/pages/Landing/components/Body/Body.module.scss:2]: assign color to variable

File: src/pages/Landing/components/Body/LeftBox/LeftBox.module.scss [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+            align-items: center;
+        }
+    }
+
+    &Active {
+        position: absolute;
+        width: 8px;
+        height: 8px;
+        border-radius: 50%;
+        background-color: #2da231;
```
Comment (ArchitW) [inline src/pages/Landing/components/Body/LeftBox/LeftBox.module.scss:92]: assign colors to vairable

File: src/pages/Teacher/Body/Assignment/Assignment.module.scss [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    margin-bottom: .125rem;
+    margin-right: .5rem;
+}
+.formCheckInput{
+    margin-right: 1rem;
+    width: 1em;
+    height: 1em;
+    margin-top: .25em;
+    vertical-align: top;
+    background-color: #fff;
```
Comment (ArchitW) [inline src/pages/Teacher/Body/Assignment/Assignment.module.scss:17]: there is color variable declared, use that

File: src/pages/Teacher/Body/Assignment/Assignment.module.scss [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+    -moz-appearance: none;
+    appearance: none;
+    -webkit-print-color-adjust: exact;
+    -webkit-border-radius: 50%;
+    -moz-border-radius: 50%;
+    -ms-border-radius: 50%;
+    -o-border-radius: 50%;
+    &:checked[type=radio],&:focus[type=radio]{
+        background-image: url('../../../../assets/images/blackdot.svg') !important;
+        background-color: #ffffff;
```
Comment (ArchitW) [inline src/pages/Teacher/Body/Assignment/Assignment.module.scss:33]: assign colors to vairable

File: src/pages/Teacher/Body/CreateAssignment/CreatAssignment.module.scss [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+
+.inputText {
+    min-height: calc(1.5em + 1rem + 5px);
+    padding: .5rem 1rem;
+    font-size: 1.5rem;
+    display: block;
+    width: 100%;
+    font-weight: 400;
+    line-height: 1.5;
+    color: #212529;
```
Comment (ArchitW) [inline src/pages/Teacher/Body/CreateAssignment/CreatAssignment.module.scss:39]: assign colors to vairable

File: src/pages/Teacher/Body/CreateAssignment/CreatAssignment.module.scss [hunk 7]
```diff
@@ -0,0 +1,106 @@
...
+    background-color: #fff;
```
Comment (ArchitW) [inline src/pages/Teacher/Body/CreateAssignment/CreatAssignment.module.scss:40]: assign colors to vairable

File: src/pages/Teacher/Body/CreateAssignment/CreatAssignment.module.scss [hunk 8]
```diff
@@ -0,0 +1,106 @@
...
+    background-clip: padding-box;
+    border: 1px solid #b1b1b1;
```
Comment (ArchitW) [inline src/pages/Teacher/Body/CreateAssignment/CreatAssignment.module.scss:42]: assign colors to vairable

File: src/pages/Teacher/Body/CreateAssignment/CreatAssignment.module.scss [hunk 9]
```diff
@@ -0,0 +1,106 @@
...
+      }
+  }
+
+  .smallInput{
+      width: 150px;
+      display: inline-block;
+      margin-right: 10px;
+  }
+  .spanC{
+    color: #fff;
```
Comment (ArchitW) [inline src/pages/Teacher/Body/CreateAssignment/CreatAssignment.module.scss:83]: assign colors to vairable

File: src/pages/Teacher/Body/CreateAssignment/CreatAssignment.module.scss [hunk 10]
```diff
@@ -0,0 +1,106 @@
...
+    background-color: #3b0e8a;
```
Comment (ArchitW) [inline src/pages/Teacher/Body/CreateAssignment/CreatAssignment.module.scss:84]: assign colors to vairable

File: src/pages/Teacher/Body/CreateAssignment/CreatAssignment.module.scss [hunk 11]
```diff
@@ -0,0 +1,106 @@
...
+    border-radius: 5px;
+    -webkit-border-radius: 5px;
+    -moz-border-radius: 5px;
+    -ms-border-radius: 5px;
+    -o-border-radius: 5px;
+}
+  .btnUp{
+      color: #fff;
+      background-color: #f16600;
+      border:1px solid  #f16600;
```
Comment (ArchitW) [inline src/pages/Teacher/Body/CreateAssignment/CreatAssignment.module.scss:96]: assign colors to vairable

File: src/pages/Teacher/Body/DropDownSelect/DropDownSelect.module.scss [hunk 12]
```diff
@@ -0,0 +1,34 @@
...
+
+.div60W{
+    display: inline-block;
+    width: 66%;
+}
+.formSelect {
+    padding-top: .5rem;
+    padding-bottom: .5rem;
+    padding-left: 1rem;
+    color: #000000;
```
Comment (ArchitW) [inline src/pages/Teacher/Body/DropDownSelect/DropDownSelect.module.scss:11]: assign colors to vairable

File: src/pages/Teacher/Body/NavTeacher/NavTaecher.module.scss [hunk 13]
```diff
@@ -0,0 +1,36 @@
...
+
+.navul {
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+    li{
+        display: inline-block;
+        padding-right: 1rem;
+            display:inline-block;
+            color: #707070;
```
Comment (ArchitW) [inline src/pages/Teacher/Body/NavTeacher/NavTaecher.module.scss:19]: assign colors to vairable

File: src/pages/Teacher/Body/NavTeacher/NavTaecher.module.scss [hunk 14]
```diff
@@ -0,0 +1,36 @@
...
+            font-size: 1.5rem;
+            font-weight:bold;
+            line-height: 1.2;
+            &:after {
+                display:block;
+                content: '';
+                border-bottom: solid 3px #f16600;
```
Comment (ArchitW) [inline src/pages/Teacher/Body/NavTeacher/NavTaecher.module.scss:26]: assign colors to vairable

**Comment by fuelupeducation** [comment]
Merged

**Pull request closed without merging**